frontend/src/metabase/visualizations/click-actions/actions/CombineColumns/CombineColumns.tsx

```diff
@@ -0,0 +1,58 @@
+import { t } from "ttag";
+
+import { CombineColumns } from "metabase/query_builder/components/expressions/CombineColumns";
```

Comment:
I think originally the popup had max width and an ellipsis for the dropdown value, and this CLS wasn't happening:

https://github.com/metabase/metabase/assets/6830683/208f24cb-12fc-41df-bf9f-0b98f4c79636

Reply:
This layout shifting became more prominent when we aligned the popup to the right.
It can happen when changing column, adding/removing a column, enabling custom separators.
IMO it would be great if this popover had a fixed width and horizontal shifting didn't occur in any of those cases.

Reply:
I've added a fixed width to the popover!